eventemitter.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+class EventEmitter {
+  constructor(){
+    this.callbacks = {};
+  }
+
+  on(event,listener){
+    this.callbacks[event] = listener;
+  }
+
+  emit(event){
+    let response = this.callbacks[event]
+    if (response === undefined){
```

Comment:
in this case you can just do this `if(!response)`, this will validate true if the content of "response" is a falsy